zingolib/src/wallet/transactions.rs

```diff
@@ -55,23 +51,28 @@ impl TransactionMetadataSet {
         mut reader: R,
         wallet_capability: &WalletCapability,
     ) -> io::Result<Self> {
-        let txs_tuples = Vector::read(&mut reader, |r| {
+        let mut witness_trees = match wallet_capability.can_spend() {
```

Comment:
Given that we require all variants to be able to spend, should we abstract a new method that's just a bool for `can_spend`.   That's how we're using it.. but with the potential for misunderstanding...

Surely in the (currently impossible, in Zingo) case of spend capabilities from some pools and viewing from others, the holder doesn't intend to not be able to spend.

It seems to me that `can_spend` is at least an inadequate name.  How about we rename it to:

`can_spend_from_any_pool`?

Reply:
Or rather..   I am going to add `can_spend_from_all_pools -> bool`
Then we should call that, when that's what we mean.

Reply:
In all instances of in-library calls, what we we're actually checking for is spend_from_all_pools.

Reply:
I see that you've added this. I like it!